API/admin/get.js

```diff
@@ -36,10 +37,20 @@ const getFlagged = async (request, response) => {
 
 const getInactive = async (request, response) => {
   try {
+    console.log("Entered with ID: ", request.params);
+
     const { id } = request.params;
-    User.findOne({ where: { id: id } }).then(row =>
+
+    await User.findOne({ where: { id: id } }).then(row =>
       response.status(200).json({ value: row.inactive })
     );
+
+    const print = await User.findOne({ where: { id: id } });
+
+    console.log(
+      "The End: ",
+      print.then(row => response.status(200).json({ value: row.inactive }))
+    );
```

Comment:
Is this needed?

Reply:
Not needed, was for testing purposes that I forgot to delete.